boot/zephyr/pm.yml

```diff
@@ -13,6 +13,15 @@ mcuboot_primary_app:
 mcuboot_primary:
   span: [mcuboot_pad, mcuboot_primary_app]
 
+#if defined(CONFIG_BOOT_SWAP_USING_MOVE)
+# This region is used to ensure that at least 1 page remains unused
+mcuboot_swap_area:
```

Comment:
Can `mcuboot_secondary_app` still share size with `mcuboot_primary_app`?

Reply:
yes that should still work.

Reply:
Sorry, I confused this one with swap using scratch method initially. Now I see it's needed.

Reply:
I am just not sure if it should be in `mcuboot_primary_app`, maybe it would be more intuitive to put it in `mcuboot_primary` (since it's not part of application I think). Still, it would work anyway, so I do not block.

Reply:
Hm, I see your point. Actually, we might end up signing a bunch of 0xFF if we put this inside `mcuboot_primary_app` (havent double checked that), I'll make a fix.